audiorecorder/src/main/java/org/odk/collect/audiorecorder/recording/AudioRecorderViewModel.kt

```diff
@@ -7,12 +7,12 @@ import java.io.File
 
 /**
  * Interface for a ViewModel that records audio. Can only record once session
```

Comment:
I think we're now in a better place with this abstraction. Adding time/amplitude helped it make more sense as it forced the key data exposed to be the current "session" details.